server/chat/backend/agent/tools/recent_incidents_tool.py

```
@@ -0,0 +1,124 @@
+                        "alert_service": row[2],
+                        "severity": row[3],
+                        "aurora_summary": summary,
+                        "started_at": row[5].isoformat() if row[5] else None,
+                        "affected_services": row[6] or [],
+                        "correlated_alert_count": row[7] or 0,
+                    })
+
+                return json.dumps({"incidents": incidents, "count": len(incidents)})
+
+    except Exception as e:
+        logger.exception("[GetRecentIncidents] Error retrieving recent incidents: %s", e)
```

Ruff (0.15.15)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)